packages/flare/python/cron_job_ingest_events.py

```diff
@@ -29,44 +33,133 @@ def main():
         raise Exception(str(e))
 
     app: client.Application = splunk_service.apps[APP_NAME]
-    flare_api = FlareAPI(app=app)
+    create_collection(app=app)
+
+    # To avoid cron jobs from doing the same work at the same time, exit new cron jobs if a cron job is already doing work
+    if get_job_running(app):
+        return
+
+    api_key: Optional[str] = None
+    tenant_id: Optional[int] = None
+    for item in app.service.storage_passwords.list():
+        if item.content.username == "api_key":
```

Comment:
Can we extract the magic strings to constants?

Maybe create an enum for the setting keys.